docs/clients/rust/index.rst

```diff
@@ -1,21 +1,41 @@
+.. _ref_rust_index:
+
 ====
 Rust
 ====
 
 :edb-alt-title: EdgeDB Rust Client
 
-EdgeDB maintains an client library for Rust. View the `full documentation
-<https://docs.rs/edgedb-tokio/latest/edgedb_tokio/>`_.
+EdgeDB maintains a client library for Rust. View the `full documentation`_.
+
+The hello world of the Rust EdgeDB client is as follows:
 
 .. code-block:: rust
 
   #[tokio::main]
-  async fn main() -> anyhow::Result<()> {
-      let conn = edgedb_tokio::create_client().await?;
-      let val = conn.query_required_single::<i64, _>(
-          "SELECT 7*8",
-          &(),
-      ).await?;
-      println!("7*8 is: {}", val);
-      Ok(())
+  async fn main() {
+      let conn = edgedb_tokio::create_client()
+          .await
+          .expect("Client should have initiated");
+      let val: i64 = conn
+          .query_required_single("select 7*8", &())
+          .await
+          .expect("Query should have worked");
+      println!("7*8 is: {val}");
   }
+
+.. _`full documentation`: https://docs.rs/edgedb-tokio/latest/edgedb_tokio/
+
+.. toctree::
+    :maxdepth: 2
+    :hidden:
+
+    getting_started
+    client
+    queryable
+    arguments
+    queryable_alternatives
```

Comment:
Should `queryable` and `queryable alternatives` be grouped together, or is `arguments` also related to the two? (The order here determines the order in the side navigation.)

Reply:
Let's see...yes, first is the call to use Queryable without any examples that have arguments, and the later queryable_alternatives section uses arguments so the page on arguments comes in between. (Personally I think it gives a bit of a mental break too)